### ISSUE TITLE
Invalid Credentials

### ISSUE THREAD

**Issue by AzodFR** [Mssjim/crunchyroll.js#2]
Hi, the Authorization Basic in getToken seems to be outdated, because my credentials stills work but got the error

```bash
 */*/node_modules/crunchyroll.js/main.js:56
     if(!client.id) throw new Error('Invalid credentials');
                          ^
Error: Invalid credentials
    at getToken (/*/*/node_modules/crunchyroll.js/main.js:56:26)
     at processTicksAndRejections (node:internal/process/task_queues:96:5)
     at async Object.login (*/*/node_modules/crunchyroll.js/main.js:75:5)
```

**Comment by Zedster07** [comment]
I'm facing the same problem , any ideas how to get or generate the Basic token?

**Comment by Mssjim** [comment]
Fixed in v1.1.0, run `npm update crunchyroll.js` to update
